src/platforms/flutter/troubleshooting.mdx

```diff
@@ -23,3 +23,9 @@ Building a Flutter app for a specific ABI using the `--target-platform` argument
 ```bash
 flutter build apk --target-platform=android-arm --split-per-abi
 ```
+
+## Native symbolication on Android
```

Comment:
maybe I also link this section to this page https://docs.sentry.io/platforms/flutter/upload-debug/#uploading-for-android-ndk
so even before uploading the files, they are gonna read this.